lib/appsignal/config.rb

```diff
@@ -622,8 +622,11 @@ def app_path
         @config.root_path
       end
 
-      def app_path=(path)
-        @config.root_path = path
+      def app_path=(_path)
+        Appsignal::Utils::StdoutAndLoggerMessage.warning \
+          "The `Appsignal.configure`'s `app_path=` writer is deprecated. " \
+            "Use the `Appsignal.configure`'s method `path` keyword argument " \
+            "to set the root path."
```

Comment:
An "is deprecated" warning implies, at least to me, that it still works as before -- this seems to no longer be the case as implemented. Let's make that explicit.

```suggestion
        Appsignal::Utils::StdoutAndLoggerMessage.warning \
          "The `Appsignal.configure`'s `app_path=` writer has been deprecated " \
            "and can no longer be used to set the root path."
            "Use the `Appsignal.configure`'s method `path` keyword argument " \
            "to set the root path."
```